### PR TITLE
Capture the YAML Template and design for CEC Emulator

### PR DESCRIPTION
**Pull request by sramani-metro**
Initial design of the emulator. Thoughts on YAML configuration.

### CODE REVIEW

**Review by Ulrond — APPROVED** [review]
minor comments

**Comment by Ulrond** [comment]
you going to merge this after updating?

File: docs/pages/hdmi_cec_emulator_design.md [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+
+```
+`./run --profile skyglass_sink_5_sources.yml --config xyz.yml`
+```
+
+A Sample profile config in YAML with a network of devices representing a TV as the root device (Sink), connected with an Audio System connected on the HDMI port 1. The Audio system has a DVR Settop on its HDMI port 1 and a chromecast connected on its port 2.
+
+```yaml
+---
+hdmicec:
```
[the remark this answers is not in the document]
Comment (sramani-metro) [reply docs/pages/hdmi_cec_emulator_design.md:19]: Done, Added comment

File: docs/pages/hdmi_cec_emulator_design.md [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+
+  power_status: &power_status
+    - on
+    - off
+    - standby
+
+  endpoint: &endpoint
+    name: !!str
+    id: *port_id
+    type: *port_type
```
[the remark this answers is not in the document]
Comment (sramani-metro) [reply docs/pages/hdmi_cec_emulator_design.md]: Done

**Review by Ulrond — APPROVED** [review]
looks good merge it

**Pull request merged**